lib/order.rb

```diff
@@ -0,0 +1,70 @@
+require './lib/menu.rb'
+
+class Order
+  attr_reader :order_list, :status
+
+  def initialize(menu) #Pass the correct Menu as argument
+    @menu = menu
+    @order_list = [] #Array containing sets of [item, quantity]
+    @status  = "not placed"
+  end
+
+  # Adds the menu item and quantity to the order
+  def add_item(item, quantity)
```

Comment:
Useful to be able to specify quantity (in mine, they'd have to add something 3 times)